scalding-core/src/main/scala/com/twitter/scalding/serialization/PositionInputStream.scala

```diff
@@ -69,8 +71,9 @@ class PositionInputStream(val wraps: InputStream) extends InputStream {
 
   override def skip(n: Long): Long = {
     if (n < 0) illegal("Must seek fowards")
-    val count = skip(n)
```

Comment:
no, we don't use it. We use skipFully, which calls the right one. Just hear to be a complete wrapper.